internal/clients/team3/judge.go

```diff
@@ -11,4 +14,139 @@ type judge struct {
 	c *client
 }
 
-// Override functions here, see president.go for examples
+// PayPresident pays the president's salary
+func (j *judge) PayPresident(salary shared.Resources) (shared.Resources, bool) {
+	j.c.params.salaryThreshold = 150 //parameter that we need to define
```

Comment:
is this the critical resources threshold?

Reply:
We intended this to be a separate threshold, such that we always wanted to have at least that amount in our local pool. I guess we can consider the critical resources threshold instead.